dotcom-rendering/index.d.ts

```diff
@@ -707,7 +714,7 @@ type FEContainerPalette =
 	| 'Breaking'
 	| 'Podcast'
 	| 'Branded'
-	| 'BreakingPalette';
+	| 'Breaking';
```

Comment:
BreakPalette was misnamed so I fixed this - though we need to revisit how palettes work as currently the /uk front has breaking as the first palette and long running alt as the second - and DCR only choses the first one